cove/input/models.py

```diff
@@ -44,6 +44,9 @@ class SuppliedData(models.Model):
 
     def get_absolute_url(self):
         return reverse('explore', args=(self.pk,), current_app=self.current_app)
+    
+    def get_raw_url(self):
```

Comment:
See my comments below, you may not need this. Instead you can pass extra arguments to ```get_absolute_url```

Reply:
Yes, this is true.